### ISSUE TITLE
Possible integrations with streamlit

### ISSUE THREAD

**Issue by hamelsmu** [fastai/nbdev#1108]
Sasha Rush built a nbdev-ish thing with Streamlit called [streambook](https://twitter.com/srush_nlp/status/1381597087079489536?s=20&t=fyegFxULZJCB_G-7GiSG_A)

This is the [GitHub repo](https://github.com/srush/streambook)

Might be interesting to revisit this and reflect if there are any additional ideas or integrations possible with nbdev based on this.  I suspect that there is.

**Comment by srush** [comment]
Yeah, I basically just use streamlit as a code viewer with jupytext. It's mostly because I like publishing in notebooks but hate coding in notebooks.

You probably could do way more with nbdev since you have access to a lot of metadata.
